source/elements/oneDAL/source/data_management/table_builders.rst

```diff
@@ -0,0 +1,58 @@
+.. _table-builders:
+
+==============
+Table builders
+==============
+
+This section contains definision of classes that implement :ref:`table-builder`
+concept.
+
+-----------------------------------
+Requirements on table builder types
+-----------------------------------
+
+Each implementation of :ref:`table-builder` concept in |dal_short_name| shall
+follow next requirements:
```

Comment:
The style is discussed in tables section
Each implementation shall:
 - provide
- be
- provide method/member function